### DISCUSSION TITLE
Vircon32 support?

### DISCUSSION THREAD

**Discussion by Michael-IRL** [OnionUI/Onion#1230] [Ideas]
Hi all, I  was just intereded in knowing if it is possiable to add vircon32 support?

**Comment by schmurtzm** [comment]
https://github.com/vircon32/ComputerSoftware#video-libraries
SDL and openGL required, I would not say impossible with what we saw recently with Drastic but probably quite difficult to build for the MM (and the fact that they don't give binaries for Rapsberry lower than version 4 is not a good sign in term of hardware requirements 😅)

**Reply by Michael-IRL** [reply]
Than you for the response, just getting into this hobby and was wondering if the 32bit fantasy console would be something that could be ported.
